frontend/src/Component/homepage.js

```diff
@@ -165,62 +167,82 @@ function HomePage() {
     <div>
       <Navbar></Navbar>
       <Container
-        mt={"xl"}
-        size={"md"}
-        my={20}
-        style={{
-          minHeight: "100vh",
-        }}
+        size={"lg"} my={20} style={{ minHeight: "80vh" }}
       >
         <div>
-          <Group position="apart">
-            <Input
-              id='name'
-              size={"sm"}
-              placeholder="Search by Name"
-              radius="lg"
-              onChange={updateSearchFields}
-              onKeyUp={search}
-              rightSection={<BsSearch />}
-            />
-            <Input
-              id='phone'
-              size={"sm"}
-              placeholder="Search by Phone"
-              radius="lg"
-              onChange={updateSearchFields}
-              onKeyUp={search}
-              rightSection={<BsSearch />}
-
-            />
-            <Input
-              id='email'
-              size={"sm"}
-              placeholder="Search by Email"
-              radius="lg"
-              onChange={updateSearchFields}
-              onKeyUp={search}
-              rightSection={<BsSearch />}
-            />
-            <Input
-              id='location'
-              size={"sm"}
-              placeholder="Search by Location"
-              radius="lg"
-              onChange={updateSearchFields}
-              onKeyUp={search}
-              rightSection={<BsSearch />}
-            />
-          </Group>
+          <Grid>
+            <Grid.Col sm={6} md={2.4} lg={1.5}>
+              <Select
+                id="pageLimit"
+                radius="lg"
+                data={pageLimits}
+                value={limit.toString()}
+                maxDropdownHeight={200}
+                onChange={
+                  (event) => {
+                    setLimit(event)
+                    setPage(1)
+                  }
+                }
+                dropdownComponent="div"
+                transition="pop-top-left"
+                transitionDuration={200}
+                transitionTimingFunction="ease" />
+            </Grid.Col>
+            <Grid.Col sm={6} md={2.4} lg={2.6}>
+              <Input
+                id='name'
+                size={"sm"}
+                placeholder="Search by Name"
+                radius="lg"
+                onChange={updateSearchFields}
+                onKeyUp={search}
+                rightSection={<BsSearch />}
+              />
+            </Grid.Col>
+            <Grid.Col sm={6} md={2.4} lg={2.6}>
+              <Input
+                id='phone'
+                size={"sm"}
+                placeholder="Search by Phone"
```

Comment:
Won't it be better to say search by number?

Reply:
It is added in a previous pr will discuss in meeting